scss/bitstyles/utilities/gap/_settings.scss

```diff
@@ -1,9 +1,14 @@
 @use '../../tools/size';
 
 $sizes: (
+  '3xs': size.get('3xs'),
+  '2xs': size.get('2xs'),
   'xs': size.get('xs'),
   's': size.get('s'),
-  'm': size.get('l'),
-  'l': size.get('xl'),
+  'm': size.get('m'),
+  'l': size.get('l'),
+  'xl': size.get('xl'),
+  '2xl': size.get('2xl'),
+  '3xl': size.get('3xl'),
```

Comment:
I wasn’t expecting to make all the sizes available as gap sizes 🤔 Would this have been useful in some past circumstances?